### PR TITLE
attribute_highlights returns an array of highlights

### PR DESCRIPTION
**Pull request by dalthon**
Just a rebase of https://github.com/toptal/chewy/pull/387

It also, instead of adding breaking change, adds a new method `*_highlights` that implements new behavior.

### CODE REVIEW

**Review by AlfonsoUceda — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
...
 
   * [#695](https://github.com/toptal/chewy/pull/695): Clear the scroll by id after completing scroll_batches ([@socialchorus][])
-  * [#749](https://github.com/toptal/chewy/pull/749): Avoid importing everything when given an empty relation ([@JF-Lalonde][][@dalthon][])
+  * [#749](https://github.com/toptal/chewy/pull/749): Avoid importing everything when given an empty relation ([@JF-Lalonde][] [@dalthon][])
   * [#736](https://github.com/toptal/chewy/pull/736): Fix nil children when using witchcraft ([@taylor-au][])
-
+  * [#755](https://github.com/toptal/chewy/pull/755): attribute_highlights returns an array of highlights ([@musaffa][] [@dalthon][])
 ## Changes
 
   * [#753](https://github.com/toptal/chewy/pull/753): Add support for direct_import parameter to skip objects reloading ([@TikiTDO][][@dalthon][])
```
Comment (rabotyaga) [inline CHANGELOG.md:11]:
```suggestion
  * [#753](https://github.com/toptal/chewy/pull/753): Add support for direct_import parameter to skip objects reloading ([@TikiTDO][], [@dalthon][])
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,9 +3,9 @@
 ## Bug fixes
 
   * [#695](https://github.com/toptal/chewy/pull/695): Clear the scroll by id after completing scroll_batches ([@socialchorus][])
-  * [#749](https://github.com/toptal/chewy/pull/749): Avoid importing everything when given an empty relation ([@JF-Lalonde][][@dalthon][])
+  * [#749](https://github.com/toptal/chewy/pull/749): Avoid importing everything when given an empty relation ([@JF-Lalonde][] [@dalthon][])
```
Comment (rabotyaga) [inline CHANGELOG.md:6]:
```suggestion
  * [#749](https://github.com/toptal/chewy/pull/749): Avoid importing everything when given an empty relation ([@JF-Lalonde][], [@dalthon][])
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,9 +3,9 @@
...
   * [#736](https://github.com/toptal/chewy/pull/736): Fix nil children when using witchcraft ([@taylor-au][])
-
+  * [#755](https://github.com/toptal/chewy/pull/755): attribute_highlights returns an array of highlights ([@musaffa][] [@dalthon][])
```
Comment (rabotyaga) [inline CHANGELOG.md:8]: Let's add new entries on the top of the corresponding section.

**Review by rabotyaga — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,13 +2,14 @@
 
 ## Bug fixes
 
+  * [#755](https://github.com/toptal/chewy/pull/755): attribute_highlights returns an array of highlights ([@musaffa][], [@dalthon][])
```
Comment (rabotyaga) [inline CHANGELOG.md:5]: Oh, and this is a kinda change, not a bug fix, right?

**Pull request merged**